src/main/resources/db/migration/0151/V187__PlantingSiteDates.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE tracking.planting_sites ADD COLUMN planting_season_start_month INTEGER;
+ALTER TABLE tracking.planting_sites ADD COLUMN planting_season_end_month INTEGER;
+
+ALTER TABLE tracking.planting_sites ADD CONSTRAINT start_month_valid
+    CHECK (planting_season_start_month BETWEEN 1 AND 12);
+ALTER TABLE tracking.planting_sites ADD CONSTRAINT end_month_valid
+    CHECK (planting_season_end_month BETWEEN 1 AND 12);
```

Comment:
if the end month is < start month, does that mean it is in the next year?

Reply:
Right. Like if your planting season is in the summer and you're in the southern hemisphere, the season might start in December and end in February.

Reply:
Thanks 👍 - makes sense.